mabl snippets/randomDropdownSelector.js

```diff
@@ -0,0 +1,12 @@
+function mablJavaScriptStep(mablInputs, callback) {
+
+  const myfunction = ()=> {
+    let select = document.getElementById('day'); //Replce 'day' with your CSS selector
+    let items = select.getElementsByTagName('option'); //Confirm your element tagnames and adjust as necessary
```

Comment:
Using `option` as well as on line 8 `select.selectedIndex = index;` means that this only works for `<select>` elements and should be stated in the comments at the top since the file name just specifies `dropdowns`

Reply:
Parameterized it.